firestore/src/android/load_bundle_task_progress_android.cc

```diff
@@ -25,7 +25,6 @@ namespace firestore {
 namespace {
 
 using jni::Class;
-using jni::Constructor;
```

Comment:
Is this change related to count queries? If not, please revert it.

Reply:
Nope, there is no backlog item for this. IMO it doesn't really matter and including it in this PR adds noise. I'm sure there are unused includes and using statements scattered everywhere in our code base!

Reply:
For small things like this, we should probably take the [Boy Scout Rule](https://github.com/97-things/97-things-every-programmer-should-know/blob/master/en/thing_08/README.md) approach. Especially, if we can't justify tracking in backlog.

Reply:
I tend to disagree with the "Boy Scout Rule" in this context. IMO PRs should be laser focused on what their purpose is. As unrelated changes bleed into a PR it makes it more difficult to review and understand after the fact. Also, if the PR needed to be reverted then those "unrelated" changes would also get reverted. You can always open a separate PR for unrelated cleanups like this and I would happily review them expeditiously. After all, I'm a huge fan of cleaning up the code base!

Reply:
I can accept excluding the change since it is in a separate file, that is otherwise untouched.

I am however concerned that we are creating unnecessary barriers that will promote tech debt. I am not about to make a separate PR to just remove a couple of imports since there is higher priority work. Which is why this barrier to be laser focused is counter productive when applied to trivial changes IMO. Changes are likely never done, and will likely create a separate cognitive burden to readers of the code.

I don't find there is a cognitive burden to pruning imports in PRs since the CI would fail if it was required. But I suppose with a mindset of being laser focused, then any deviation creates questions and cognitive load.

I do find unused imports create a burden to the reader of code. I was looking for an example of how to use `jni::Constructor` in our code base, and this import was a red herring (among others) that made my work with the code slower.